src/data/history.ts

```diff
@@ -230,6 +230,10 @@ export const computeHistory = (
       unit = hass.config.unit_system.temperature;
     } else if (computeStateDomain(stateInfo[0]) === "humidifier") {
       unit = "%";
+    } else if (computeStateDomain(stateInfo[0]) === "counter") {
```

Comment:
Side note:
We should compute the domain once, instead of in every if statement...